setup.py

```diff
@@ -104,6 +104,9 @@ def get_long_description():
 
 
 if __name__ == '__main__':
+    # Force "make develop" inside READTHEDOCS environment
+    if os.environ.get("READTHEDOCS") and "install" in sys.argv:
+        os.system("make develop")
```

Comment:
BTW, the commit message is a bit misleading: this doesn't change the Makefile (even though it runs it).